Header header
float64 angle
float64 scan_radius
uint8[] scanline_data
